Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/BoardUI.cs

```diff
@@ -1,8 +1,43 @@
 ﻿using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 
 public class BoardUI : MonoBehaviour 
 {
+    public Board _Board;
+    HashSet<Animator> _animatingTiles;
+
+    public void Init()
+    {
+        _animatingTiles = new HashSet<Animator>();
+    }
+
+    public void PlayerPrompt_MovableTiles(HashSet<TileData> reachableTiles)
+    {
+        //for each of the tiles set animator variable 
+        Animator nextAnimator;
+        foreach (var tile in reachableTiles)
+        {
+            //grab animator
+            nextAnimator = tile.TileObject.GetComponentInChildren<Animator>();
+
+            //if already animating then reset it
+            if (nextAnimator.GetTime() > 0)
+                nextAnimator.SetTime(0);
+
+            nextAnimator.SetBool("CanBeMovedTo", true);
+            _animatingTiles.Add(nextAnimator);
+        }
+    }
+
+    public void PlayerPrompt_DefualtTiles()
```

Comment:
Just noticed the misspelling, it's up to you whether you want to fix now or follow on later